src/Benchmarks/Data/EfDb.cs

```diff
@@ -83,7 +83,7 @@ public async Task<IEnumerable<Fortune>> LoadFortunesRows()
                 result.Add(fortune);
             }
 
-            result.Add(new Fortune { Message = "Additional fortune added at request time." });
+            result.Add(new Fortune { Message = "Additional fortune added at request time."u8.ToArray() });
```

Comment:
Does it reuse the array, or should it be declared as a class member.

Reply:
> Within the scope of the request, a new Fortune object must be constructed and added to the list. This confirms that the data structure is dynamic-sized. The new fortune is not persisted to the database; it is ephemeral for the scope of the request.

It's a little vague on the details, but I feel like it should be created per request.

Reply:
Can make it a member like https://github.com/aspnet/Benchmarks/blob/05ad4582116b62443cefe5bef4086bfe159d02a2/src/BenchmarksApps/TechEmpower/PlatformBenchmarks/Data/RawDb.cs#L291

Reply:
I guess a hardcoded string isn't created per request either.